tools/WinObjC.Logging/WinObjC.Logging.targets

```diff
@@ -8,25 +8,57 @@
          NOTE: It is important that this information is added at the end in the .targets instead of in the .props so that consumers don't
          accidentally stomp it. This is contrary to a configurable type property that a consumer may want / need to adjust. To depend on
          this package means the consumer gets these these things.  -->
+    <ItemGroup Condition="'$(ConfigurationType)' != 'StaticLibrary'">
+      <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.winmd"/>    
+      <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.winmd"/>
+
+      <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\*.winmd"/>
```

Comment:
Yes its basically supposed to be a generic one. This is all supposed to be handled by the packaging package. Didn't we have a task to use it in the main build?